content/preferences-settings/general.md

```diff
@@ -27,12 +27,16 @@ GUI controls and text DPI
 
 # CSS Theme Modifications
 
-In addition to selecting a pre-built theme you can also apply additional CSS customisations of your own to tweak the look-and-feel of darktable. A text box is provided for this purpose.
+In addition to selecting a pre-built theme you can also apply additional CSS customisations of your own to tweak the look-and-feel of darktable. darktable provides a textbox for this purpose in general tab of main preferences. No need to set import lines of darktable CSS themes. You just need to choose which default darktable theme you want to start from (in theme menu just above) and then just copy/paste tweaks you find or create your own.
 
-When you have finished entering your CSS tweaks, click on the 'save CSS and apply' button. This will save your CSS to a file (`$HOME/.config/darktable/user.css`) and immediately apply it to the current darktable session.
+When you have finished entering your CSS tweaks, click on the 'save CSS and apply' button. This will save your CSS to a file (`$HOME/.config/darktable/user.css` on Mac and Linux ; `C:\%LOCALAPPDATA%\darktable\user.css` on Windows) and immediately apply it to the current darktable session.
```

Comment:
I don't use windows much but is this location always on the C drive? Or is there a more generic thing (like $HOME) that we can use. It's worth noting that this will be the only place we reference windows file locations so, for consistency it might be better to stay Linux/Mac-only here and update them all at once as a separate change.

Reply:
Or perhaps even provide a single section about file locations and just link to it from everywhere.

Reply:
By default, it should be only on C: drive and as far as I know, there's no more generic thing that what I write here. I follow some instructions provided by Windows user.
About a specific section, probably better but this section should be set first. This text could be updated later when that section will be done. It's a request of this issue: #171